cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -133,6 +143,24 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				workloadmeta.Module(),
 				fx.Provide(tagger.NewTaggerParams),
 				tagger.Module(),
+				fx.Supply(
+					status.Params{
+						PythonVersionGetFunc: func() string { return python.GetPythonVersion() },
+					},
+					status.NewInformationProvider(collectorStatus.Provider{}),
+					status.NewHeaderInformationProvider(net.Provider{}),
+					status.NewInformationProvider(leaderelection.Provider{}),
+					status.NewInformationProvider(clusteragentMetricsStatus.Provider{}),
+					status.NewInformationProvider(admissionpkg.Provider{}),
+					status.NewInformationProvider(endpointsStatus.Provider{}),
+					status.NewInformationProvider(autodiscoveryStatus.Provider{}),
+					status.NewInformationProvider(clusterchecks.Provider{}),
+					status.NewInformationProvider(orchestratorStatus.Provider{}),
+				),
+				// The cluster agent displays log agent informaton if "compliance_config.enabled"
```

Comment:
The cluster-agent wants to get `logs.Status`, but the `statusProvider` is in the same package as `comp/logs/agent`, and the cluster-agent doesn't need all of the logs-agent. My first thought is that `comp/logs/agent/status.go` should move to `comp/logs/status/status.go`, but I haven't looked into how big of a refactor that would be or what the consequences are. At first glance it seems correct: `comp/logs/bundle.go` would change the `Bundle()` to include `agent.Module()` and `status.Module()`. The cluster-agent could include the `status.Module()` but not the `agent.Module()`.